README.md

```diff
@@ -25,6 +25,9 @@ and zone. It supports the following fields:
 | Option | Type | Description |
 | ------ | ---- | ----------- |
 | `credentials` | `String` | A path to the service account credentials file. Accepts either an absolute path or a path relative to the Bolt project directory. _Optional._ |
+| `gcloud_client_email` | `String` | The Google Cloud client email to use.
+| `gcloud_token_uri` | `String` | The Google Cloud token uri to use.
+| `gcloud_private_key` | `String` | The Google Cloud private key to use.
```

Comment:
I don't think we want the `gcloud_` prefix on these. I added `aws_` for the aws version because the keys in the credentials file have that prefix, so I figured that's the canonical form. It looks like the gcloud credentials file just calls them `client_email`, `token_uri`, `private_key`.

Reply:
Fair enough. I'll update that